pkg/storage/user_accountStore.go

```diff
@@ -98,6 +103,30 @@ func (s *Store) UserAccountGetByIDLOADER(
 	return userSlice, nil
 }
 
+// UserAccountNotificationPreferencesNewModelPlan gets the notification
+// preferences for a new model plan for a slice of users who have at least one
+// enabled preference
+func (s *Store) UserAccountNotificationPreferencesNewModelPlan(np sqlutils.NamedPreparer) (
+	[]*models.UserAccountNotificationPreferences,
+	error,
+) {
+
+	var userNotificationPreferences []*models.UserAccountNotificationPreferences
```

Comment:
No need to refactor this unless you want to, but there are a couple examples of a new paradigm that abstracts away some boiler plate work for named statements (`sqlutils.SelectProcedure`). See  `user_notification_preferencesStore.go` for examples.

Reply:
I intended to get to this on Monday, but now facing time constraints and am trying to get backend work for another feature out this week. Though in the same vein as another comment, I suspect this function will be deprecated in favor of a more generic solution after some tech debt is addressed.